### PR TITLE
include integritee parachain

### PR DESCRIPTION
**Pull request by brenzi**
the linter currently complains, but not about my changes

### CODE REVIEW

File: packages/apps-config/src/ui/logos/index.ts [hunk 1]
```diff
@@ -69,6 +70,7 @@ export const chainLogos: Record<string, unknown> = [
   ['Galois', nodeMath],
   ['HydraDX Hydrate', chainHydrate],
   ['Encointer PC1', nodeEncointerNotee],
+  ['IntegriTEE PC1', nodeIntegritee],
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:73]:
```suggestion
  ['IntegriTEE PC1', nodeIntegritee],
```

**Comment by jacogr** [comment]
I did apply the linting fix (trailing whitespace).

**Comment by brenzi** [comment]
cheers. I actually called yarn lint but failed to commit that auto-fix.

I still get `tokenDecimals.map is not a function` in the browser. Don't know why

**Comment by jacogr** [comment]
If running locally, you are probably just missing a `yarn` to install the latest deps.

**Comment by jacogr** [comment]
Thank you!

**Pull request merged**